OpenRA.Mods.Common/Activities/Execute.cs

```diff
@@ -0,0 +1,34 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using OpenRA.Activities;
+
+namespace OpenRA.Mods.Common.Activities
+{
+	public class Execute : Activity
+	{
+		Action f;
```

Comment:
We used to have an activity like this, called 'CallFunction'. It was nuked from orbit.